packages/cache-utils/package.json

```
@@ -5,9 +5,9 @@
   "type": "module",
   "exports": "./lib/main.js",
   "main": "./lib/main.js",
-  "types": "./lib.main.d.ts",
+  "types": "./lib/main.d.ts",
   "files": [
-    "lib/**/*.js"
+    "lib/**/*.{js,ts}"
```

I think it would be nice to be a bit more specific here, since we don't want actual TypeScript sources files to make their way into the npm package, just the declaration files.

```suggestion
    "lib/**/*.{js,d.ts}"
```

---

The `lib` folder only had the built output from `tsc`, but yeah it makes sense to limit it to d.ts